### PR TITLE
py sol for elevator rides

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
AC:
Screenshot from 2024-09-13 22-02-55
Screenshot from 2024-09-13 22-03-03

File: solutions/gold/cses-1653.mdx [hunk 1]
```diff
@@ -132,4 +132,46 @@ int main() {
 }
 ```
 </CPPSection>
+
+<PySection>
+
+```py
+people, maxWeight = map(int, input().split())
```
Comment (SansPapyrus683) [inline solutions/gold/cses-1653.mdx:139]: python uses snake_case

File: solutions/gold/cses-1653.mdx [hunk 2]
```diff
@@ -132,4 +132,46 @@ int main() {
...
+dp = [(people + 1, maxWeight + 1)] * (1 << people)
+dp[0] = (1, 0)
+
+"""
+Loop through all bitmasks.
+The bitmasks represent whether each person has used the elevator or not.
+If the ith bit is set, this means the ith person has used the elevator.
+"""
+
+for mask in range(1, (1 << people)):
```
Comment (SansPapyrus683) [inline solutions/gold/cses-1653.mdx:151]: unnecessary parens

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/gold/cses-1653.mdx (hunk 1 above)
Comment (freakin23) [reply solutions/gold/cses-1653.mdx:139]: Sorry, it just slips

**Review by ryanchou-dev — APPROVED** [review]
address kevin's comments first, but yeah looks good

**Comment by freakin23** [comment]
AC:
Screenshot from 2024-09-14 10-29-33
Screenshot from 2024-09-14 10-29-39

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**